test/parallel/test-path-parse-format.js

```diff
@@ -25,20 +25,27 @@ const assert = require('assert');
 const path = require('path');
 
 const winPaths = [
-  'C:\\path\\dir\\index.html',
-  'C:\\another_path\\DIR\\1\\2\\33\\\\index',
-  'another_path\\DIR with spaces\\1\\2\\33\\index',
-  '\\foo\\C:',
-  'file',
-  '.\\file',
-  'C:\\',
-  '',
+  // [path, root]
+  ['C:\\path\\dir\\index.html', 'C:\\'],
+  ['C:\\another_path\\DIR\\1\\2\\33\\\\index', 'C:\\'],
+  ['another_path\\DIR with spaces\\1\\2\\33\\index', ''],
+  ['\\foo\\C:', '\\'],
```

Comment:
`path` does not do validation, but it's not:
![image](https://user-images.githubusercontent.com/96947/28524777-e38b39d6-704f-11e7-9901-3db6b06904d7.png)